src/nvim/version.c

```diff
@@ -80,8 +80,8 @@ static int included_patches[] = {
   784,
   //783,
   //782,
-  //781,
-  //780,
+  781,
+  //780 NA
```

Comment:
Sorry for my poor English, Maybe you misunderstand what i'm meaning. what I say is clarify the reason in your pull request message, not remove it, mark patches as NA is welcome to add to another pr(at least I think so). 
What you need is explain it in your pr message (a brief reason and a link to the NA patch, that's all.
Sorry another time for don't explain myself well.